video/out/cocoa_common.m

```diff
@@ -615,11 +570,13 @@ void vo_cocoa_set_opengl_ctx(struct vo *vo, CGLContextObj ctx)
 int vo_cocoa_config_window(struct vo *vo)
 {
     struct vo_cocoa_state *s = vo->cocoa;
+    struct mp_vo_opts *opts  = vo->opts;
+
     run_on_main_thread(vo, ^{
-        struct mp_rect screenrc;
-        vo_cocoa_update_screen_info(vo, &screenrc);
         vo_cocoa_update_screen_fps(vo);
 
+        NSRect r = [s->current_screen frame];
+        struct mp_rect screenrc = (struct mp_rect){0, 0, r.size.width, r.size.height};
```

Comment:
This is an initializer, so you don't actually need the `(struct mp_rect)` part (although it's still allowed).